msticpy/data/drivers/odata_driver.py

```diff
@@ -184,12 +184,8 @@ def connect(
                 authority=authority,
                 username=cs_dict["username"],
                 scopes=self.scopes,
-                auth_type=kwargs["auth_type"]
-                if "auth_type" in kwargs
-                else "interactive",
-                location=cs_dict["location"]
-                if "location" in cs_dict
-                else "token_cache.bin",
+                auth_type=kwargs.get("auth_type", "interactive"),
```

Comment:
Is interactive a good default vs. devicecode? I thought this didn't work in AML or anywhere with a remote kernel

Reply:
Thats a good point - i will swap for devicecode